hq/conf/routes

```diff
@@ -9,11 +9,12 @@ GET        /iam                                       controllers.HQController.i
 GET        /iam/:accountId                            controllers.HQController.iamAccount(accountId)
 GET        /security-groups                           controllers.SecurityGroupsController.securityGroups
 GET        /security-groups/:accountId                controllers.SecurityGroupsController.securityGroupsAccount(accountId)
+GET        /snyk                                      controllers.SnykController.snyk
 GET        /login                                     controllers.AuthController.login
 GET        /loginError                                controllers.AuthController.loginError
 GET        /oauthCallback                             controllers.AuthController.oauthCallback
 GET        /logout                                    controllers.AuthController.logout
-GET        /documentation/                            controllers.UtilityController.documentation(file = "index")
+GET        /documentation                             controllers.UtilityController.documentation(file = "index")
```

Comment:
Will this work? I tried dropping this `/` before but it seemed to break the linking to the subpages once on the documentation page...

Reply:
Yes, it works; I did it to bring into line with all the others.

However I failed to update the header....  hold on.

Reply:
Reverted.